spec/forms/submission_form_spec.rb

```diff
@@ -107,13 +111,38 @@
     end
 
     describe 'on save' do
-      before { submission } # do this first to get accurate tare of record counts
+      context 'with valid params' do
+        before { submission } # do this first to get accurate tare of record counts
+
+        it 'creates new records for itself and all nested objects' do
+          expect { submission.save }
+            .to  change(Location,   :count).by(1)
+            .and change(Listing,    :count).by(1)
```

Comment:
should there be another test where it's 1 submission and 2+ listings?

Reply:
Same as above.